packages/common/src/utils/formatUtil.ts

```diff
@@ -175,3 +175,27 @@ export const parseWeiNumber = (number: string) => {
     return null
   }
 }
+
+type FormatOptions = {
+  minDecimals?: number
+  maxDecimals?: number
+  excludeCommas?: boolean
+}
```

Comment:
spaces plz :)

Reply:
we should figure out how to encode this as an ESLint rule 👀